### DISCUSSION TITLE
Add reject() function for wallets.

### DISCUSSION THREAD

**Discussion by chudilka1** [TenKeyLabs/dappwright#215] [Q&A]
Is there a chance to get a reject() function added for `Wallet` that would reject the transactions (the opposite of approve())?

**Comment by osis** [comment]
Yes! Should be an easy addition and I wonder why it hasn't come up before. 🤔

Should be a very easy PR for anyone to contribute. 🙌

**Comment by alejogranerorequest** [comment]
Hey @osis , could you develop it? It is very useful

**Reply by osis** [reply]
Will take a look at it during the next slot of time. Should be an easy one for anyone to pick up though if it's urgent.

**Comment by osis** [comment]
implemented in https://github.com/TenKeyLabs/dappwright/pull/280
